src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/Sdk/AutoImport.props

```diff
@@ -22,7 +22,7 @@ https://github.com/dotnet/designs/blob/4703666296f5e59964961464c25807c727282cae/
     <Using Include="Android.OS.Bundle" Alias="Bundle" Platform="Android" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(EnableDefaultAndroidItems)' == 'true' and $([MSBuild]::VersionEquals($(TargetFrameworkVersion), '8.0')) ">
+  <ItemGroup Condition=" '$(MonoAndroidResourcePrefix)' != '' and '$(EnableDefaultAndroidItems)' == 'true' and $([MSBuild]::VersionEquals($(TargetFrameworkVersion), '8.0')) ">
```

Comment:
Given that XML attributes *can* contain newlines, at what point -- if an -- should this become a multi-line expression?  It's kinda long right now…

Reply:
Yeah, putting it on multiple lines would be OK. What should that look like? How is it indented?